composer.json

```diff
@@ -23,19 +23,23 @@
     "require": {
         "php": "^7.2"
     },
+    "conflict": {
+        "doctrine/mongodb-odm": "<1.3"
+    },
     "require-dev": {
         "doctrine/dbal": "^2.5",
+        "doctrine/mongodb-odm": "^2.0",
```

Comment:
I think we should give some consistency to this dependency across all our repositories, sometimes we have an special build on github actions for this (but only for version 1) other times we use 1.3 || 2.0 on composer and here we only have 2.0

Reply:
I agree, IMHO we could drop support for 1.x, 
- `1.x` uses the deprecated `mongodb` PHP extension.
- we can get rid of `alcaeus/mongo-php-adapter`.
- `2.0` was release 1 year ago
- `2.0` requires minimum PHP 7.2 while `1.x` requires minimum of 5.6

Reply:
Can you take care of updating the rest of repositories, and dev-kit?

Reply:
> I agree, IMHO we could drop support for 1.x,
> 
> * `1.x` uses the deprecated `mongodb` PHP extension.

Yeah ! Since I don't have this extension, I can't install project with this dependency :)

Reply:
> Can you take care of updating the rest of repositories, and dev-kit?

Sure, let me make PRs, I'll also remove the variant in the PR to check that everything is ok.